src/screens/Evaluation/EvaluationScreen.js

```diff
@@ -1,58 +1,86 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import {
   StyleSheet, View,
 } from 'react-native';
-import { Button } from 'react-native-elements';
-import PropTypes from 'prop-types';
+// import { Button } from 'react-native-elements';
+// import PropTypes from 'prop-types';
 import Firestore from '@react-native-firebase/firestore';
 import EvalVideo from './EvalVideo';
 import TimestampedFeedbackList from './TimestampedFeedbackList';
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    justifyContent: 'center',
+    justifyContent: 'flex-end',
     alignItems: 'center',
   },
-  formContainer: {
-    height: 400,
-    padding: 20,
+  bottomContainer: {
+    width: '100%',
+    height: '50%',
   },
-  comment: {
-    flex: 1,
-    flexDirection: 'column',
-    justifyContent: 'flex-start',
-    alignContent: 'flex-start',
-    marginTop: 5,
-    marginLeft: 3,
-    marginBottom: 5,
-    marginRight: 3,
+  timestampedFeedback: {
   },
 });
 
-export default function EvaluationScreen({ navigation }) {
-  const [evaluation, setEvaluation] = useState({});
+export default function EvaluationScreen() {
+  const videoPlayer = useRef(null);
+
+  const [evaluationDoc, setEvaluationDoc] = useState({});
+  const [evaluations, setEvaluations] = useState([]);
+  const [evaluationsLoaded, setEvaluationsLoaded] = useState(false);
+  const [seekUntil, setSeekUntil] = useState({});
+  const [videoLinkLoaded, setVideoLinkLoaded] = useState(false);
+  const [videoLink, setVideoLink] = useState();
+
   const docId = 'bTzLmdl03mDOYwsZMyCP';
```

Comment:
i think next task would be to have the doc id be passed in either as part of the navigation, or as part of the functional component itself